charts/tempo-distributed/values.yaml

```diff
@@ -0,0 +1,363 @@
+  # -- Number of replicas for the query-frontend
+  replicas: 1
+  image:
+    # -- The Docker registry for the query-frontend image. Overrides `tempo.image.registry`
+    registry: null
+    # -- Docker image repository for the query-frontend image. Overrides `tempo.image.repository`
+    repository: null
+    # -- Docker image tag for the query-frontend image. Overrides `tempo.image.tag`
+    tag: null
+  # -- The name of the PriorityClass for query-frontend pods
+  priorityClassName: null
+  # -- Annotations for query-frontend pods
+  podAnnotations: {}
+  # -- Additional CLI args for the query-frontend
+  extraArgs: []
+  # -- Environment variables to add to the query-frontend pods
+  extraEnv: []
+  # -- Environment variables from secrets or configmaps to add to the query-frontend pods
+  extraEnvFrom: []
+  # -- Resource requests and limits for the query-frontend
+  resources: {}
+  # -- Grace period to allow the query-frontend to shutdown before it is killed
+  terminationGracePeriodSeconds: 30
+  # -- Affinity for query-frontend pods. Passed through `tpl` and, thus, to be configured as string
+  # @default -- Hard node and soft zone anti-affinity
+  affinity: {}
+    # podAntiAffinity:
+    #   requiredDuringSchedulingIgnoredDuringExecution:
+    #     - labelSelector:
+    #         matchLabels:
+    #           {{- include "tempo.queryFrontendSelectorLabels" . | nindent 10 }}
+    #       topologyKey: kubernetes.io/hostname
+    #   preferredDuringSchedulingIgnoredDuringExecution:
+    #     - weight: 100
+    #       podAffinityTerm:
+    #         labelSelector:
+    #           matchLabels:
+    #             {{- include "tempo.queryFrontendSelectorLabels" . | nindent 12 }}
+    #         topologyKey: failure-domain.beta.kubernetes.io/zone
+  # -- Node selector for query-frontend pods
+  nodeSelector: {}
+  # -- Tolerations for query-frontend pods
+  tolerations: []
+
+
+config: |
+  auth_enabled: false
+  compactor:
+    compaction:
+      block_retention: 48h
+    ring:
+      kvstore:
+        store: memberlist
+  distributor:
+    receivers:
+      jaeger:
+        protocols:
+          grpc:
+            endpoint: 0.0.0.0:14250
+          thrift_binary:
+            endpoint: 0.0.0.0:6832
+          thrift_compact:
+            endpoint: 0.0.0.0:6831
+          thrift_http:
+            endpoint: 0.0.0.0:14268
+  querier:
+    frontend_worker:
+      frontend_address: {{ include "tempo.queryFrontendFullname" . }}:9095
+  ingester:
+    lifecycler:
+      ring:
+        replication_factor: 1
+  memberlist:
+    abort_if_cluster_join_fails: false
+    join_members:
+      - {{ include "tempo.fullname" . }}-gossip-ring
+  overrides:
+    per_tenant_override_config: /conf/overrides.yaml
+  server:
+    http_listen_port: 3100
+  storage:
+    trace:
+      backend: local
+      blocklist_poll: 0
+      local:
+        path: /var/tempo/traces
+      pool:
+        max_workers: 10
+        queue_depth: 100
+      wal:
```

Comment:
some of these fields have moved in the upcoming 0.6.0 version of tempo.  this will probably be published early next week:

```
storage:
  trace:
    block:
      bloom_filter_false_positive: 0.0.1
      index_downsample_bytes: 2_000_000
    cache:  memcached
```

https://github.com/grafana/tempo/blob/master/CHANGELOG.md